winui/BusyIndicator/Content.md

```diff
@@ -1,12 +1,80 @@
 ---
 layout: post
-title: Content with WinUI BusyIndicator control | Syncfusion
-description: Learn here all about showing the loading progress of the content with Syncfusion WinUI BusyIndicator Control.
+title: Content in WinUI BusyIndicator control | Syncfusion
+description: Learn here all about the loading progress content and its customization available in the Syncfusion WinUI BusyIndicator control.
```

Comment:
description: Learn all about the loading progress content and its customization available in the Syncfusion WinUI BusyIndicator control here.

Reply:
Changed the description content as mentioned above.